target/linux/ramips/image/mt7621.mk

```diff
@@ -753,6 +753,17 @@ define Device/iptime_a8004t
 endef
 TARGET_DEVICES += iptime_a8004t
 
+define Device/iptime_a3004nd
+  $(Device/dsa-migration)
+  $(Device/uimage-lzma-loader)
```

Comment:
Just curious, are you sure you need this? What happens if you omit this?

Reply:
Hello, OpenWrt does not boot without the $(Device/uimage-lzma-loader).
According to a UART log the router gets lzma error and reboots. 
About $(Device/dsa-migration) : I think this was just copied from iptime-a8004t. I will try removing it and testing.

Reply:
Okay, then other iptime devices might need that remedy. Thanks for the info.
AFAIK you need to keep `$(Device/dsa-migration)` part.